src/Groups/action.jl

```diff
@@ -453,6 +454,45 @@ end
 
 on_subgroups(x::T, g::GAPGroupElem) where T <: GAPGroup = T(on_subgroups(GapObj(x), g))
 
+
+@doc raw"""
+    on_rref_mats(m::MatElem{T}, x::MatrixGroupElem) where T <: FinFieldElem
+
+Return the image of `m` under `x`,
+where the action is given by first computing the product `m * x`
+and then normalizing the result by computing its reduced row echelon form,
+see [`rref(M::MatrixElem{T})`](@ref).
+
+Identifying `m` with the subspace of the natural module for the group of `x`
+that is generated by the rows of `m`,
+`on_rref_mats` describes the action on subspaces of this natural module.
+Note that for computing the orbit and stabilizer of `m` w.r.t. `on_rref_mats`,
+`m` must be in reduced row echelon form.
+
+# Examples
+```jldoctest
+julia> n = 3;  q = 2;  F = GF(q);  V = free_module(F, n);
+
+julia> G = GL(n, F);
+
+julia> W, embW = sub(V, [gen(V,1), gen(V,3)])
+(Subspace over F with 2 generators and no relations, Hom: W -> V)
+
+julia> m = matrix(embW)
+[1   0   0]
+[0   0   1]
+
+julia> S, _ = stabilizer(G, m, on_rref_mats);  order(S)
+24
+
+julia> orb = orbit(G, on_rref_mats, m);  length(orb)
+7
+```
+"""
+function on_rref_mats(m::MatElem{T}, x::MatrixGroupElem) where T <: FinFieldElem
+  return rref(m * x)[2]
```

Comment:
```suggestion
  return echelon_form(m * x)
```

Reply:
It is the same, but since we have `echelon_form`, we might as well make use of it.